tests/integration/conftest.py

```diff
@@ -185,6 +185,19 @@ def set_up_requirements(self):
         self.add_keys_and_certificates()
         self.add_client_tools_repo()
         self.install_package()
+        if set_to_stage:
+            self.set_up_to_stagecdn()
+
+    def set_up_to_stagecdn(self):
+        # Point the server hostname to the staging environment,
+        # so we don't need to pass it to convert2rhel explicitly
+        # RHSM baseurl gets pointed to a stage cdn
+        self.shell(
+            "subscription-manager config --rhsm.baseurl=https://{} --server.hostname={}".format(
+                TEST_VARS["RHSM_STAGECDN"], TEST_VARS["RHSM_SERVER_URL"]
```

Comment:
Just for clarity
```suggestion
            "subscription-manager config --rhsm.baseurl=https://{0} --server.hostname={1}".format(
                TEST_VARS["RHSM_STAGECDN"], TEST_VARS["RHSM_SERVER_URL"]
```

Reply:
Thanks, I'll add it for better readability

Reply:
done